src/main/java/org/broadinstitute/consent/http/models/support/TicketFields.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.consent.http.models.support;
+
+import java.util.List;
+import org.broadinstitute.consent.http.enumeration.SupportRequestType;
+
+public record TicketFields(String name, SupportRequestType type, String email, String subject,
+                           String description, String url, List<String> uploads) {
+
+  public void validate() {
+    if (name() == null || email() == null) {
+      throw new IllegalArgumentException("Name and email of user requesting support is required");
+    }
+    if (subject() == null) {
+      throw new IllegalArgumentException("Subject is required");
+    }
+    if (description() == null) {
+      throw new IllegalArgumentException("Description is required");
+    }
+    if (type() == null) {
+      throw new IllegalArgumentException("Type is required");
+    }
+    if (url() == null) {
+      throw new IllegalArgumentException("URL is required");
+    }
```

Comment:
References to fields from inside a `record` don't need to use method calls. Also the common code here could be refactoreed
```suggestion
  private static void validate(boolean condition, String prefix) {
    if (!condition) {
      throw new IllegalArgumentException(prefix + " is required");
    }
  }

  public void validate() {
    validate(name == null || email == null, "Name and email of user requesting support");
    validate(subject == null, "Subject");
    validate(description == null, "Description");
    validate(type == null, "Type");
    validate(url == null, "URL");
  }
```